data_management/management_utils/management_utils.py

```diff
@@ -83,15 +84,18 @@ def get_file_structure(self) -> (list[PrePipelineStoryRef],str):
             else:
                 print("error: invalid structure, no story found: ", check_path)
 
-        return valid_story_paths,self.data_path
+        return valid_story_paths, self.data_path
 
 
 if __name__ == "__main__":
     CDL = CPUDataLoader()
-    file_structure=CDL.get_file_structure()
+    file_structure = CDL.get_file_structure()
+    print(file_structure)
     ref_dataset = PrePipelineDatasetRef(*file_structure)
-    ids = ref_dataset.get_tf_dataset()
-    print(ids)
-    #print(ref_dataset)
-    #for d in ref_dataset.story_refs:
-    #    print(d)
+    print(ref_dataset)
+    for a in ref_dataset.story_refs:
+        print(a)
+    #tf_dataset = ref_dataset.get_tf_dataset()
```

Comment:
any reason not to not just delete these lines?

Reply:
No reason now, they can be deleted. Originally I needed the information for another feature I was working on. This is what we get for me concentrating on too many different projects at once!